### ISSUE TITLE
No way to force next action to wait on terminal command completion

### ISSUE THREAD

**Issue by tmcarr** [fifafu/BetterTouchTool#444]
**What kind of Mac are you using?**
15" Macbook Pro with TouchBar

**Which input device (e.g. MacBook Trackad, Magic Mouse, Magic Trackpad Touch Bar) is affected by this?**
Touchbar

**Which versions of macOS and BetterTouchTool are you running?**
Mac OS: 10.12.4 Beta (16E154a)
BTT: 2.02

**Steps to repeat the problem:**

1. Create a new button, call it whatever you want. In my case: "test"
2. for the action select terminal command
3. Configure the command to be "sleep 1000000"
4. Click add additional action button
5. Set it to run an applescript
6. Set applescript to: "display notification "This should take a while""

**Expected Behavior:**
It takes a LONG TIME before the notification appears

**Actual Behavior:**
The notification appears immediately, implying the sleep is still running in the background

**Comment by bennyty** [comment]
I have the same need setup as hotkey → sqlite query | pbcopy → ⌘V → sqlite delete. However sometimes the paste happens before pbcopy resolves.

**Comment by tmcarr** [comment]
Wow.... github's versioning is pretty swanky! 

Props to you guys, never seen such an efficiently managed issue system on github.

**Comment by fifafu** [comment]
Sorry for not replying earlier to this issue. I'm currently cleaning up here, which is why I'm closing some older issues. In case you still have questions or problems, please create a new issue - I'm now much more responsive than a few months ago.

The next alpha of BTT will have two versions of the "Run Terminal Command" action. One synchronous (blocking) and one asynchronous.
